resources/views/components/game/devbox-claim-management.blade.php

```diff
@@ -84,7 +84,7 @@
 
 <script>
 function makeClaim() {
-    const gameTitle = "{!! html_entity_decode($gameTitle) !!}";
+    const gameTitle = "{!! addslashes($gameTitle) !!}";
```

Comment:
Since `gameTitle` is no longer sanitized, we have to do an alternate escaping. In reality, we only need to escape double quotes here.